src/layouts/ultimate-guide.tsx

```diff
@@ -28,21 +34,47 @@ const UltimateGuideLayout: FunctionComponent<LayoutProps> = ({
           title,
           description,
           url,
-          images: ogImage ? [ogImage] : undefined,
+          images: ogImage
+            ? [ogImage]
+            : [
+                {
+                  url: `https://og-image-react-egghead.now.sh/article/Codemods%20with%20Babel%20Plugins?author=${
```

Comment:
default share card looks like this: <a href="https://og-image-react-egghead.now.sh/article/Codemods%20with%20Babel%20Plugins?author=Laurie%20Barth"><img src="https://og-image-react-egghead.now.sh/article/Codemods%20with%20Babel%20Plugins?author=Laurie%20Barth" width="250"></a>